### ISSUE TITLE
Subtitle support

### ISSUE THREAD

**Issue by burnoutberni** [screeninvader/ScreenInvader#22]
mpv comes with excellent subtitle support built in, but there's no way to access it with the ScreenInvader.

One should be able to:
- Switch subtitle channel (either if the video file comes with subtitles or if subtitles have been added manually)
- Add subtitle channel (from URL, file and maybe even automatically from OpenSubtitles.org)
- Sync subtitles (+/- in case they aren't perfectly in sync with the video channel)
- more??

**Comment by kallaballa** [comment]
i agree. though manually adding a subtitle is a bit fiddley to implement and at the moment i don't have time to implement it. maybe in a month or two.
